tools/logger/logger.c

```diff
@@ -150,9 +150,9 @@ static int append_filter_config(struct convert_config *config, const char *input
 
```

Comment:
typo: filet?

Reply:
fixed

Reply:
do you mean filter instead of filer? Or am I missing something :D?

Reply:
Speedy fingers 😸